packages/web-components/src/components/cbp-app-header/cbp-app-header.stories.tsx

```diff
@@ -23,17 +23,27 @@ const Template = ({ context, sx }) => {
         ${sx ? `sx=${JSON.stringify(sx)}` : ''}
       >
         <a slot="cbp-home" href="/" class="nav-home">Application Name</a>
-        <cbp-app-header-nav 
-          variant='multi'
-          text='Multi Nav Item'
-          href='/'
-        >
-        </cbp-app-header-nav>
-        <cbp-app-header-nav
-          text='Single Nav Item'
-          href='/'
-        >
-        </cbp-app-header-nav>
+        <cbp-nav-item>
```

Comment:
I'd make these an array of objects and pass them in as an argument. Use # for the href so that it stays on the same page and can demo updating the selected/current state.

Reply:
updated to use the arguments and hooked the up to the storybook controls